mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/UnderlineStyle.kt

```diff
@@ -8,7 +8,7 @@ import kotlin.jvm.JvmInline
 
 @Immutable
 @JvmInline
-public value class UnderlineStyle private constructor(
+public value class UnderlineStyle internal constructor(
```

Comment:
Worth commenting that this cannot be private due to the bug?

Reply:
Sure. I did also change the build configuration so it will fail if you try.